pkg/get/download.go

```diff
@@ -31,6 +32,24 @@ func Download(tool *Tool, arch, operatingSystem, version string, downloadMode in
 		return "", "", err
 	}
 
+	// Check user provided version is valid on dry run
+	if len(version) > 1 && dry_run {
+		res, err := http.Head(downloadURL)
+		if err != nil {
+			return "", "", err
+		}
+
+		if res.StatusCode != http.StatusOK {
+			return "", "", fmt.Errorf("incorrect status for downloading tool: %d", res.StatusCode)
```

Comment:
I'm not sure that we want "incorrect status", this should indicate that the tool is not available or not found.